ui-tests/src/test/java/io/syndesis/qe/steps/other/ConcurValidationSteps.java

```diff
@@ -0,0 +1,86 @@
+package io.syndesis.qe.steps.other;
+
+import cucumber.api.java.en.When;
+import io.syndesis.qe.utils.HttpUtils;
+import lombok.extern.slf4j.Slf4j;
+import okhttp3.FormBody;
+import okhttp3.Headers;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import org.json.JSONObject;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+/**
+ * Maybe this will become handy in the future when we receive correct credentials for both
+ * OAuth and WS and both will point to the same instance. For now this class is not used.
+ */
+@Slf4j
+public class ConcurValidationSteps {
+    private static String ACCESS_TOKEN = null;
+    private static final String ACCESS_TOKEN_ENDPOINT = "https://us-impl.api.concursolutions.com/oauth2/v0/token";
+    // you can find which endpoint you need and how to use it here:
+    // https://developer.concur.com/api-reference/expense/expense-report/reports.html
+    private static final String CONCUR_ENDPOINT_EXPENSE_REPORTS = "https://implementation.concursolutions.com/api/v3.0/expense/reports";
+
+    @When("^check that concur list or whatever we need to test is here \"([^\"]*)\"$")
+    public void invokeRequest(String something) {
```

Comment:
not used?

Reply:
Correct, this whole class is not used, you can see why in class javadoc. I don't want to remove it as it works and in the future we may receive both WS and OAuth credentials pointing to the same instance.

Reply:
imo commiting something that is not used anywhere is not a good practice

Reply:
Well, I also thought about not including this verification file and just leave it in my branch if required later... Will do.

Reply:
looks better :dancer: